lib/lookup.json

```diff
@@ -106,7 +106,7 @@
   "eslint": {
     "prefix": "v",
     "flaky": ["v7", "s390", "ubuntu"],
-    "skip": ["v8", "ppc"],
+    "skip": ["v8", "ppc", ">9"],
```

Comment:
could this be `[">v8", "ppc"],?

Reply:
I guess it should be `[">8", "ppc"]`? I am not sure what labels all exist but I doubt that the `v` is necessary.